main.JS

```diff
@@ -0,0 +1,22 @@
+'use strict';
+
+function Fixed(){
```

Comment:
- javascript は `camelCase` が一般的なので、先頭は小文字にしましょう。
- `fixed` という名前だと「何をしてくれる関数なのか」いまいちわかりにくいです
  - `outputFixedLogs` なんていかがでしょう？

```suggestion
function outputFixedLogs(){
```